tests/unit/CorrectBusinessContactInfo.spec.ts

```diff
@@ -0,0 +1,129 @@
+// Libraries
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import VueRouter from 'vue-router'
+import mockRouter from './MockRouter'
+import sinon from 'sinon'
+import { axios } from '@/utils'
+
+// Store
+import { getVuexStore } from '@/store'
+
+// Components
+import { createLocalVue, mount } from '@vue/test-utils'
+import { BusinessContactInfo } from '@/components/YourCompany'
+import flushPromises from 'flush-promises'
+
+Vue.use(Vuetify)
+const localVue = createLocalVue()
+localVue.use(VueRouter)
+const router = mockRouter.mock()
+const vuetify = new Vuetify({})
+
+const contactInfo = {
+  email: 'mock@email.com',
+  confirmEmail: 'mock@email.com',
+  phone: '250-123-4567'
+}
+
+describe('BusinessContactInfo for a correction', () => {
+  let wrapper: any
+  let store: any = getVuexStore()
+
+  const originalCorrectionContact = {
+    email: 'mockCorrection@email.com',
+    confirmEmail: 'mockCorrection@email.com',
+    phone: '250-123-4567'
+  }
+
+  beforeAll(async () => {
+    await router.push({ name: 'correction' })
+    store.state.stateModel.defineCompanyStep.businessContact = contactInfo
+    store.state.stateModel.originalIA.incorporationApplication.contactPoint = originalCorrectionContact
+  })
+
+  beforeEach(async () => {
+    wrapper = mount(BusinessContactInfo, { vuetify, store, localVue, router })
+  })
+
+  afterEach(() => {
+    wrapper.destroy()
+  })
+
+  it('renders the CorrectBusinessContactInfo Component', async () => {
+    expect(wrapper.find(BusinessContactInfo).exists()).toBe(true)
+  })
+
+  it('loads the correct original contact info for a correction', async () => {
+    expect(wrapper.vm.originalContact.email = originalCorrectionContact.email)
+    expect(wrapper.vm.originalContact.phone = originalCorrectionContact.phone)
```

Comment:
You should be able to use `expect(blah).toEqual()`.

Reply:
Updated!